ocs_ci/utility/oc_utils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from semantic_version import Version
+
+from ocs_ci.utility.utils import exec_cmd
+
+log = logging.getLogger(__name__)
+
+
+def get_client_version():
+    """
+    Get the major.minor version of the openshift client
+
+    Returns:
+        str: major.minor oc version
+
+    """
+    # get version info from oc
+    cmd = "oc version"
+    resp = exec_cmd(cmd, ignore_error=True)
+    stdout = resp.stdout.decode()
+    # get the full version from the output
+    full_version_str = stdout.split(":")[1].strip()
+    log.debug("full version string: %s", full_version_str)
+    # remove the date from the version string
+    version = "-".join(full_version_str.split("-")[:2])
```

Comment:
How it will behave for released version? e.g.:
```
$ oc version
Client Version: 4.6.9
Server Version: 4.7.0-0.nightly-2021-03-25-035231
Kubernetes Version: v1.20.0+bafe72f
```

Reply:
I tested this case and because the way I'm using slices still works for the version returned for GA versions. However, I'm going to change this to use the json output like you suggested so it will most likely be more consistent anyways.